lib/loaders/typescript.js

```diff
@@ -32,6 +32,17 @@ module.exports = {
             [config]
         );
 
+        // fork-ts-checker-webpack-plugin integration
+        if (webpackConfig.useForkedTypeScriptTypeChecking) {
+            loaderFeatures.ensureLoaderPackagesExist('forkedtypecheck');
+            // force transpileOnly to speed up
+            config.transpileOnly = true;
+
+            // add forked ts types plugin to the stack
+            const forkedTypesPluginUtil = require('../plugins/forked-ts-types');
+            forkedTypesPluginUtil(webpackConfig);
+        }
```

Comment:
What about logging a recommendation here to enable this feature for faster type-checking? We now have a `logger.js` module that can be used for this (though I might add a new method on that - `recommendation()` - instead of using `debug()`.